### PR TITLE
Release ehrenamtskarte without ausweisen

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: frontend/lib/home/home_page.dart [hunk 1]
```diff
@@ -27,6 +28,16 @@ class _HomePageState extends State<HomePage> {
   @override
   void initState() {
     super.initState();
+    final List<AppFlow> additionalTabs = buildConfig.featureFlags.verification
```
Comment (f1sh1918) [inline frontend/lib/home/home_page.dart:31]: i think it should just be called `verificationTab`

**Review by f1sh1918 — APPROVED** [review]
tested on ios

File: frontend/lib/home/home_page.dart [hunk 2]
```diff
@@ -43,12 +54,7 @@ class _HomePageState extends State<HomePage> {
         "Suche",
         GlobalKey<NavigatorState>(debugLabel: "Search tab key"),
       ),
-      AppFlow(
-        const IdentificationPage(title: "Ausweisen"),
-        Icons.remove_red_eye_outlined,
-        "Ausweisen",
-        GlobalKey<NavigatorState>(debugLabel: "Auth tab key"),
-      ),
+      ...additionalTabs,
```
Comment (michael-markl) [inline frontend/lib/home/home_page.dart:57]: Can't you do an if clause inside the array declaration? 
Would be a lot easier to read.
https://github.com/dart-lang/language/blob/master/accepted/2.3/control-flow-collections/feature-specification.md

**Review by michael-markl — APPROVED** [review]
Would be nice if we could fix https://github.com/digitalfabrik/entitlementcard/issues/794 before releasing.

File: frontend/lib/home/home_page.dart (hunk 2 above)
Comment (maxammann) [reply frontend/lib/home/home_page.dart:57]: I google for this feature but failed 😂

**Pull request merged**